.github/workflows/backend.yml

```
@@ -0,0 +1,43 @@
+        run: |
+          docker build -t $IMAGE_NAME:${{ github.sha }} -f server/Dockerfile .
+      - name: Push image
+        run: |
+          docker push $IMAGE_NAME:${{ github.sha }}
+      - name: Tag latest (optional)
+        if: github.ref == 'refs/heads/scout/sco-1-d886e992'
+        run: |
+          docker tag $IMAGE_NAME:${{ github.sha }} $IMAGE_NAME:latest
+          docker push $IMAGE_NAME:latest
+      - name: Optional Render deploy
+        if: ${{ secrets.RENDER_DEPLOY_HOOK != '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)